featuretools/tests/entityset_tests/test_serialization.py

```diff
@@ -31,7 +26,7 @@
 BUCKET_NAME = "test-bucket"
 WRITE_KEY_NAME = "test-key"
 TEST_S3_URL = "s3://{}/{}".format(BUCKET_NAME, WRITE_KEY_NAME)
-TEST_FILE = "test_serialization_data_entityset_schema_5.1.0.tar"
+TEST_FILE = "test_serialization_data_entityset_schema_5.2.0.tar"
```

Comment:
if we're unable to read in the 5.1.0 test file successfully we need to move to 6.0.0

if so, this would also impact the feature serialization tests

Reply:
Yeah, we can't read because of switching terminology from `entity` to `dataframe` and `variable` to `column`. I'll update to 6.0.0 instead.